src/pages/BeerDetails.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link, NavLink } from "react-router-dom";
+
+function BeersList(props) {
+  const { beers } = props;
+  return (
+    <div>
+      {beers.map((beer) => {
```

Comment:
Here you will only display the info of one beer so you don't need to loop over it